src/js/components/FileInput/FileInput.js

```diff
@@ -130,133 +130,206 @@ const FileInput = forwardRef(
         rightOffset =
           removeRef.current.getBoundingClientRect().width +
           rightPad.replace('px', '');
-      else rightOffset = removeRef.current.getBoundingClientRect().width;
+      if (files.length === 1) {
+        rightOffset =
+          controlRef.current.getBoundingClientRect().width +
+          removeRef.current.getBoundingClientRect().width +
+          24;
```

Comment:
Since I have `margin="small"` on `anchor` I have a hardcoded `24` but if margin is the right direction we want to go I can change this to calculate margin

Reply:
I don't think we should hardcode. instead should use `theme.edgeSize.small` or wherever the "margin small" value comes from

Reply:
I agree just left it hardcoded for the time beings until I was sure which theme.margin we would use! 👍